File: 3.cpp
#include "ptoc.h"


real a,f;
int main(int argc, const char* argv[])
{
pio_initialize(argc, argv);
output << "a=";input >> a >> NL;
if(a<-1) 
f=sqr(a);
else if ((a>=1) && (a<=2)) 
f=sqr(a);
else
f=4;
output << "f=" << format(f,5,2) << NL;
input >> NL;
return EXIT_SUCCESS;
}

File: 3.pas
program p3;
var a,f:real;
begin
write ('a=');readln(a);
if(a<-1) then
f:=sqr(a)
else if (a>=1) and (a<=2) then
f:=sqr(a)
else
f:=4;
writeln('f=',f:5:2);
readln;
end.